crates/re_viewer/src/ui/time_panel/mod.rs

```diff
@@ -136,38 +140,35 @@ impl TimePanel {
     fn collapsed_ui(&mut self, ctx: &mut ViewerContext<'_>, ui: &mut egui::Ui) {
         ui.spacing_mut().item_spacing.x = 18.0; // from figma
 
-        ctx.rec_cfg
-            .time_ctrl
-            .time_control_ui(ctx.re_ui, ctx.log_db.times_per_timeline(), ui);
-
-        {
-            let mut time_range_rect = ui.available_rect_before_wrap();
-            time_range_rect.max.x -= 220.0; // save space for current time
-
-            if time_range_rect.width() > 50.0 {
-                let time_ranges_ui =
-                    initialize_time_ranges_ui(ctx, time_range_rect.x_range(), None);
-                time_ranges_ui.snap_time_control(ctx);
-
-                let painter = ui.painter_at(time_range_rect.expand(4.0));
-                painter.hline(
-                    time_range_rect.x_range(),
-                    time_range_rect.center().y,
-                    ui.visuals().widgets.inactive.fg_stroke,
-                );
-                time_marker_ui(
-                    &time_ranges_ui,
-                    &mut ctx.rec_cfg.time_ctrl,
-                    ui,
-                    &painter,
-                    &time_range_rect,
-                );
-
-                ui.allocate_rect(time_range_rect, egui::Sense::hover());
-            }
+        if ui.max_rect().width() < 600.0 {
```

Comment:
consider using `is_mobile` (or `is_portrait_mode`?) as established in re_ui. It's nice to have a hard size threshold in a single place, this is already a different width here and further below

Reply:
We need to support a range of devices, not just "mobile" and "desktop", so it makes sense to have different limits for different parts of the UI. This limit (600px) is very much specific for the time panel.